File: oop/oop.js
class Attendee {
    constructor(name, ticket) {
        this.name = name;
        this.ticket = ticket;
    }
    displayAttendee() {
        console.log(`Name : ${this.name}, Ticket : ${this.ticket}`)
    }
    addTicket() {
        this.ticket += 1
        console.log(`${this.name} tickets increades to ${this.ticket}`)
    }
}

const attendee1 = new Attendee('M. Villani', 2)
const attendee2 = new Attendee('N. Solazzo', 1)

attendee2.addTicket()
attendee2.addTicket()

attendee1.displayAttendee()
attendee2.displayAttendee()

attendee2.responseTicket = function(){
    console.log(`This is response of my ticket: ${this.ticket}`)
}

attendee1.secondName = 'M. Ferguson'
attendee1.displayAttendeeNewName = function() {
    console.log(`Name : ${this.secondName}, Ticket : ${this.ticket}`)
}

attendee1.displayAttendeeNewName()
attendee2.responseTicket()

File: oop/oop.py
class Attendee:
    
    def __init__(self, name, ticket):
        self.name = name
        self.ticket = ticket

    def displayAttendee(self):
        print('Name : {}, Ticket : {}'.format(self.name, self.ticket))
    
    def addTicket(self):
        self.ticket += 1
        print('{} tickets increades to {}'.format(self.name, self.ticket))
    
attendee1 = Attendee('M. Villani', 2)
attendee2 = Attendee('N. Solazzo', 1)

attendee2.addTicket()
attendee2.addTicket()

attendee1.displayAttendee()
attendee2.displayAttendee()